.github/workflows/test.yaml

```
@@ -149,10 +149,10 @@ jobs:
 
       - name: Install Chrome stable binary
         run: |
-          sudo apt-get update -qq \
-          && sudo curl -o "/tmp/google-chrome-stable_current_amd64.deb" --no-clobber "https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb" \
+          sudo apt-get update -qq 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)